CODE_GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Code Standards and Style Guide
+
+## Format
+
+Our base code style is [PEP8](https://www.python.org/dev/peps/pep-0008/), but we allow a max line length of 120.
+
+Check formatting and dead code with
+
+```shell
+make lint
+```
+
+## Code Standards
+
+### Context Use
+
+We make use of the [Behave Context](https://behave.readthedocs.io/en/stable/tutorial.html#context) object to store data
+that is needed across multiple steps of a scenario.
+
+To prevent the context from becoming cluttered and confusing to use we define some rules for how we interact with it:
+
+- Only step functions and environment hooks should interact with the context attributes
+- Context attributes should be documented in the [Context Index](#Context Index)
+
+#### Context Index
+
+Every context attribute used by the tests should be described here.
+
+| Attribute                 | Description                                                                                     |
+| ------------------------- | ----------------------------------------------------------------------------------------------- |
+| test_start_local_datetime | Stores the local time at the beginning of each scenario in an environment hook                  |
+| survey_id                 | Stores the ID of the survey generated and or used by the scenario                               |
+| collex_id                 | Stores the ID of the collection exercise generated and or used by the scenario                  |
+| emitted_cases             | Stores the collectionCase DTO objects emitted on `CASE_CREATED` events                          |
+| emitted_cases_id          | Stores the case IDs from the emitted `CASE_CREATED` events                                      |
```

Comment:
Although added by my good self, I think emitted_cases_id isn't really required.  
Is there any difference between emitted_cases_id[0] & emitted_cases[0]['caseId']  

Again less on the context, less complexity

Reply:
I've quickly tested this locally, took 10 mins to retire  emitted_cases_id